gensim/utils.py

```diff
@@ -1378,8 +1378,8 @@ def unpickle(fname):
         Python object loaded from `fname`.
 
     """
-    with smart_open(fname, 'rb') as f:
-        # Because of loading from S3 load can't be used (missing readline in smart_open)
+    with open(fname, 'rb') as f:
+        # Because of loading from S3 load can't be used (missing readline in open)
```

Comment:
This comment also incorrect.

Unrelated: @mpenkov is this comment true? Is `smart_open` missing some methods? I don't understand this code block… what's up with the `latin1`?

Reply:
OK, looking at the [Python pickle docs](https://docs.python.org/3/library/pickle.html#pickle.load), this seems related to compatibility for loading Python 2 pickles in Python 3.

```
Optional keyword arguments are fix_imports, encoding and errors, which are used to control compatibility support for pickle stream generated by Python 2. If fix_imports is true, pickle will try to map the old Python 2 names to the new names used in Python 3. The encoding and errors tell pickle how to decode 8-bit string instances pickled by Python 2; these default to ‘ASCII’ and ‘strict’, respectively. The encoding can be ‘bytes’ to read these 8-bit string instances as bytes objects. Using encoding='latin1' is required for unpickling NumPy arrays and instances of datetime, date and time pickled by Python 2.
```

Still not sure what this comment line is trying to say about S3 and smart_open though.

Reply:
fixed